### ISSUE TITLE
WCAG examples are missing

### ISSUE THREAD

**Issue by tobsn** [h5bp/html5-boilerplate#177]
just because i couldnt find anything else to bother you guys:
http://www.w3.org/WAI/

in most EU countries all the gov websites got completely redesigned/recoded between 2002 and 2006 to support strong accessibility guidelines (WCAG1.0) to comply a disability law which says that everyone has the same right, which also means to be able to freely surf the internet.

example css/html and JS might be interesting for some people.
infos: http://www.w3.org/WAI/WCAG20/quickref/

fyi - america has 39 million disabled individuals and some of them would probably be glad to hear about it ;)
